### PR TITLE
Update README.md to include Noto Sans Ethiopic

### LINKED ISSUE #56: Add Ge'ez (Ethiopic) script fonts
Under ["Africa Middle East"](https://github.com/satbyy/go-noto-universal#go-noto-africa-middle-east), the languages listed are:
> Adlam, Anatolian Hieroglyphics, Arabic (Naskh-style), Avestan, Bamum, Bassa Vah, Chorasmian, Cuneiform, Egyptian, Elymaic, Hatran, Hebrew, Imperial Aramaic, Inscriptional Pahlavi, Inscriptional Parthian, Mandaic, Manichaean, Meroitic, Nabataean, Old North Arabian, Old Persian, Old South Arabian, Palmyrene, Phoenician, Psalter Pahlavi, Samaritan, Syriac, Ugaritic, Yezidi.

This doesn't include the Ge'ez script for Ethiopic languages.
The relevant Noto fonts for [serif](https://fonts.google.com/noto/specimen/Noto+Serif+Ethiopic) and [sans-serif](https://fonts.google.com/noto/specimen/Noto+Sans+Ethiopic) are linked and are each an additional 566 glyphs, which should fit within the 65,535 unicode font glyph limit given the reported size of `GoNotoCurrent.ttf` .

### PR DESCRIPTION
**Pull request by quantum9Innovation**
Close #56

### CODE REVIEW

**Comment by satbyy** [comment]
Can you please add the other missing names also? For example, all these are missing in README even though they are implemented in the font:

>   "NotoSansEthiopic-Regular.ttf"
>    "NotoSansOsmanya-Regular.ttf"
>   "NotoSansTifinagh-Regular.ttf"
>    "NotoSansNKo-Regular.ttf"
>    "NotoSansVai-Regular.ttf"
>    "NotoSansMendeKikakui-Regular.ttf"
>    "NotoSansMedefaidrin-Regular.ttf"

File: README.md [hunk 1]
```diff
@@ -6,12 +6,12 @@ ancient) or geographical region (Africa-MiddleEast, Europe-Americas, South Asia,
 
 We offer two types of combined fonts:
 
-1. Time-based:  
+1. Time-based:
```
Comment (satbyy) [inline README.md:9]: Can you please keep the two trailing spaces as they are? It forces a line break in Markdown.

Same comment on line 14.

**Review by satbyy — CHANGES_REQUESTED** [review]
Thanks for cross-checking the chapters! This is a valuable contribution!

File: README.md (hunk 1 above)
Comment (quantum9Innovation) [reply README.md]: Sorry totally forgot about that; I was trying to remove any trailing spaces that might have been created.
It should be fixed now!

**Review by satbyy — APPROVED** [review]
Thanks. Look good.

**Pull request merged**